web/packages/teleterm/src/services/config/configService.ts

```diff
@@ -168,14 +200,28 @@ function getDefaultTerminalFont(platform: Platform) {
   }
 }
 
-export function createConfigService(
-  appConfigFileStorage: FileStorage,
-  platform: Platform
-) {
-  return createConfigStore(
-    createAppConfigSchema(platform),
-    appConfigFileStorage
-  );
+function getKeyboardShortcutDescription(about: string): string {
+  return `Shortcut to ${about}. A valid shortcut contains at least one modifier and a single key code, for example "Ctrl+Shift+A". Function keys do not require a modifier.`;
+}
+
+export function createConfigService({
+  configFile,
+  configJsonSchemaFile,
+  platform,
+}: {
+  configFile: FileStorage;
+  configJsonSchemaFile: FileStorage;
```

Comment:
`configFile` and `schemaFile` alone would be fine I think. We're in `createConfigService`, the schema cannot be for anything other than the config so I don't feel like a prefix is necessary. ;)

Reply:
Maybe `jsonSchemaFile`? We already have `schema` variable (it's a Zod schema). Adding a `json` prefix makes it more clear that these two schemas are different things.